lambda-notes/src/Components/rightContent/MyNotes.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const MyNotes = (props) => {
+	return (
+		<div>
+			{props.notes.map((note) => {
+				return (
+					<NoteDetails key={note._id} title={note.title} textBody={note.textBody} _id={note._id} {...props} />
+				);
+			})}
+		</div>
+	);
+};
+
+const NoteDetails = (props) => {
```

Comment:
I like this organization with a simple app like this one. It's often our instinct to make a new file for every component but I think you were right that you didn't quite need that for the cards list.